docs/cunumeric/source/comparison/_comparison_generator.py

```diff
@@ -131,9 +163,7 @@ def _section(header, mod_ext, other_lib, klass=None, exclude_mod=None):
         "   Number of NumPy functions: {}".format(len(base_funcs)),
         "   Number of functions covered by "
         f"{other_lib}: {len(lg_funcs & base_funcs)}",
-        "   {} specific functions:".format(other_lib),
     ]
-    buf += ["   - {}".format(f) for f in (lg_funcs - base_funcs)]
```

Comment:
the same goes here. if this is a separate improvement you want to make, please make a separate pr.